README.md

```diff
@@ -51,14 +51,18 @@ A task with the following resources:
 ```
       resources:
         limits:
-          nvidia.com/gpu: 1 # requesting 1 vGPU
-          nvidia.com/gpumem: 3000 # Each vGPU contains 3000m device memory
+          nvidia.com/gpu: 1 # Declare how many physical GPUs the pod needs
```

Comment:
In common practice, inline comments start with a lowercase letter, while standalone comments begin with an uppercase letter.

such as: https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer
https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors